lib/MusicBrainz/Server/Edit/Medium/SetTrackLengths.pm

```diff
@@ -133,6 +133,14 @@ sub accept {
         );
     }
 
+    my $cdtoc_id = $self->data->{cdtoc}{id};
+    if (!$self->c->model('CDTOC')->get_by_id($cdtoc_id)) {
+        MusicBrainz::Server::Edit::Exceptions::FailedDependency->throw(
+            'The CD TOC the track times were being set from no longer exists. '.
+            'It may have been removed since this edit was entered.'
+        );
```

Comment:
Since this will presumably be left as a note, can we use `localized_note` here?

(I also wonder what other cases there are besides it being removed that we have to say it "may" have been removed.)

Reply:
I'm not sure! And because I wasn't sure, I wrote "may". But it might very well be that it is the only possibility.